manoseimas/settings/development.py

```diff
@@ -38,6 +38,14 @@
 
 INSTALLED_APPS += (
     'debug_toolbar',
+    'webpack_loader',
 )
 
 DATABASES['default']['NAME'] = 'manoseimas'
+
+WEBPACK_LOADER = {
```

Comment:
I think webpack should just handle everything if you pass `-p` flag to it when building production bundle(s).